Library/Formula/camlp5.rb

```diff
@@ -13,20 +13,41 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-tex", "Install the pdf, ps, and tex documentation"
 
   depends_on "ocaml"
+  depends_on "gnu-sed" => :build if build.with?("doc") || build.with?("tex")
+  depends_on :tex => [:build, :optional]
+  depends_on "ghostscript" => :build if build.with?("tex") && !File.exist?("/usr/local/bin/ps2pdf")
 
   def install
-    if build.with? "strict"
-      strictness = "-strict"
-    else
-      strictness = "-transitional"
+    if build.with?("tex")
+      ps2pdf = which("/usr/local/bin/ps2pdf") # known MacTex installation path
+      ps2pdf ||= which(Formula["ghostscript"].bin/"ps2pdf")
+      odie "Sorry can't find your ps2pdf from MacTex's or Homebrew's GhostScript" if ps2pdf.nil?
```

Comment:
> How do you feel about making your own GhostScript keg_only since it conflicts directly with MacTeX, which is the main way people satisfy the :tex requirement

Note that doing so would involve revisions to every `brew uses ghostscript` formula that hasn't been version/revision updated since October the 16th 2015, the day [this commit](https://github.com/Homebrew/homebrew/commit/3b0cbe6a56d9133941482a9e2d033ad86cdb5e79) was merged, and a revision to `ghostscript` to remove the symlinks in `HOMEBREW_PREFIX`. It wouldn't necessarily be a small deal to get done.